e2e/suite1/specs/trace-view-scrolling.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('Trace view', () => {
 
     e2e.pages.Explore.General.scrollBar().scrollTo('center');
 
-    // After scrolling we should have 140 spans instead of the first 100
-    e2e.components.TraceViewer.spanBar().should('have.length', 140);
+    // After scrolling we should have 139 spans instead of the first 100
+    e2e.components.TraceViewer.spanBar().should('have.length', 139);
```

Comment:
Seems like the changes in padding/margins changes how much rows is loaded on scroll. @zoltanbedi maybe there is a better fix for this?